site/en/docs/extensions/whatsnew/index.md

```diff
@@ -3,7 +3,7 @@ layout: 'layouts/doc-post.njk'
 title: What's new in Chrome extensions
 description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
 date: 2021-02-25
-updated: 2023-05-17
+updated: 2023-05-25
```

Comment:
@AmySteam, do we usually bump this for spelling fixes?

Reply:
I don't think so